awslambdaric/lambda_literals.py

```diff
@@ -0,0 +1,15 @@
+"""
+Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Holds warning message that is emitted when an unhandled exception is raised during function invocation.
```

Comment:
The file intention would be to store messages constants. Probably the comment belongs t line between 5 and 6